pkg/agent/route/route_linux.go

```diff
@@ -116,11 +119,12 @@ type Client struct {
 }
 
 // NewClient returns a route client.
-func NewClient(networkConfig *config.NetworkConfig, noSNAT, proxyAll, connectUplinkToBridge bool) (*Client, error) {
+func NewClient(networkConfig *config.NetworkConfig, noSNAT, proxyAll, connectUplinkToBridge, multicastEnabled bool) (*Client, error) {
```

Comment:
I think the function "NewClient" in route_windows.go also needs to change.

Reply:
updated